src/opm/output/eclipse/RestartIO.cpp

```diff
@@ -505,10 +504,10 @@ void writeHeader(ecl_rst_file_type * rst_file,
   }
 
 
-void writeExtraData(ecl_rst_file_type* rst_file, const std::map<std::string,std::vector<double>>& extra_data) {
-    for (const auto& pair : extra_data) {
-        const std::string& key = pair.first;
-        const std::vector<double>& data = pair.second;
+  void writeExtraData(ecl_rst_file_type* rst_file, const RestartValue::extra_vector& extra_data) {
```

Comment:
Extra spaces?

Reply:
Ehhh - where?

Reply:
Two spaces at the start of the line light up in GitHub for me at least.